File: front/src/dto/kanrensha/getKanrenshaPoliOrgCapsuleDto.ts
import type MasterPoliticalOrganizationInterface from "../../entity/masterPoliticalOrganizationEntity";
import MasterPoliticalOrganizationEntity from "../../entity/masterPoliticalOrganizationEntity";

export default interface GetKanrenshaPoliOrgCapsuleInterface {

}


export default class GetKanrenshaPoliOrgCapsuleDto implements GetKanrenshaPoliOrgCapsuleInterface {


    /** 関連者政治団体マスタEntity */
    masterPoliticalOrganizationEntity: MasterPoliticalOrganizationInterface;

    constructor() {
        this.masterPoliticalOrganizationEntity = new MasterPoliticalOrganizationEntity();
    }
}

File: back/src/main/java/mitei/mitei/political/balancesheet/manage/kanrensha/dto/user/GetKanrenshaPoliOrgCapsuleDto.java
package mitei.mitei.political.balancesheet.manage.kanrensha.dto.user;

import java.io.Serializable;

import mitei.mitei.political.balancesheet.manage.kanrensha.dto.FrameworkCapsuleDto;
import mitei.mitei.political.balancesheet.manage.kanrensha.entity.MasterPoliticalOrganizationEntity;

/**
 * 関連者政治団体格納Dto
 */
public class GetKanrenshaPoliOrgCapsuleDto extends FrameworkCapsuleDto implements Serializable {

    /** Serialize id */
    private static final long serialVersionUID = 1L;

    /** 関連者政治団体マスタEntity */
    private MasterPoliticalOrganizationEntity masterPoliticalOrganizationEntity = new MasterPoliticalOrganizationEntity();

    /**
     * 関連者政治団体マスタEntityを取得する
     *
     * @return 関連者政治団体マスタEntity
     */
    public MasterPoliticalOrganizationEntity getMasterPoliticalOrganizationEntity() {
        return masterPoliticalOrganizationEntity;
    }

    /**
     * 関連者政治団体マスタEntityを設定する
     *
     * @param masterPoliticalOrganizationEntity 関連者政治団体マスタEntity
     */
    public void setMasterPoliticalOrganizationEntity(
            final MasterPoliticalOrganizationEntity masterPoliticalOrganizationEntity) {
        this.masterPoliticalOrganizationEntity = masterPoliticalOrganizationEntity;
    }

}
